Lib/test/test_ttk/test_widgets.py

```diff
@@ -952,25 +975,26 @@ def test_configure_value(self):
 
 @unittest.skipIf(sys.platform == 'darwin',
                  'ttk.Scrollbar is special on MacOSX')
-@add_standard_options(StandardTtkOptionsTests)
+@add_configure_tests(StandardTtkOptionsTests)
 class ScrollbarTest(AbstractWidgetTest, unittest.TestCase):
     OPTIONS = (
         'class', 'command', 'cursor', 'orient', 'style', 'takefocus',
     )
+    _rounds_pixels = False
+    _clipped = {}
     default_orient = 'vertical'
 
     def create(self, **kwargs):
         return ttk.Scrollbar(self.root, **kwargs)
 
-
-@add_standard_options(PixelSizeTests if tk_version >= (8, 7) else IntegerSizeTests,
+@add_configure_tests(PixelSizeTests if tk_version >= (8, 7) else IntegerSizeTests,
```

Comment:
I left quite a few tests against (8, 7) in place.  There are many more than just this.  They are not wrong.  They are just useless, assuming that no one ever tries to support using 8.7 with tkinter.  Our understanding is that 8.7 will never be released, so that  assumption is presumably valid.  I guess I could go through all files and replace (8,7) by (9,0) if that seems worth doing.